api/v1beta2/network_types.go

```diff
@@ -107,6 +177,38 @@ type ClassicELB struct {
 	Tags map[string]string `json:"tags,omitempty"`
 }
 
+// NLBSpec defines an AWS network load balancer.
+type NLBSpec struct {
```

Comment:
Rename this to LB. All LoadBalancers in v2 have these settings, so it makes sense to generalise it.